okhttp/src/test/java/okhttp3/internal/ws/RealWebSocketTest.java

```diff
@@ -350,6 +351,63 @@
         250d));
   }
 
+  @Test public void writesUncompressedMessageIfCompressionDisabled() throws IOException {
+    server.initWebSocket(random, 0, false, false);
+
+    server.webSocket.send(ByteString.encodeUtf8("Hello"));
+    server.webSocket.writeOneFrame$okhttp();
```

Comment:
Second issue here. 

The 4 tests verify that the actual data is compressed `if (contextTakeover || compressedDataSize < rawDataSize)`. 
Therefore, we need to see the data from the buffer _after_ it is written and _before_ it is read. Calling `writeOneFrame` manually is the only way I found that catches the data in this intermediate state. 

The (minor?) issue with this is that IntelliJ raises a soft error (IDE-level, compilation not affected): 
![Screen Shot 2019-07-19 at 19 39 00](https://user-images.githubusercontent.com/14860034/61551106-efe53580-aa5c-11e9-83bb-a7113ad5e5b6.png)

Not sure if it's a bug or not, but I think that it should not be a blocker.